app/models/job.rb

```diff
@@ -1,6 +1,6 @@
 class Job < ActiveRecord::Base
   belongs_to :project
-  belongs_to :user
+  belongs_to :user, -> { unscope(where: 'deleted_at') }
```

Comment:
unscoped doesn't work.  It works as a class method to remove all scopes.  In associations unscoped doesn't seem to work at all, and only the query methods seems to work.